tests/testAutopas/tests/autopasInterface/IteratorTest.cpp

```diff
@@ -289,21 +293,30 @@ void IteratorTest::testOpenMPIterators(autopas::ContainerOption containerOption,
 #ifdef AUTOPAS_OPENMP
 #pragma omp parallel
 #endif
-  for (auto iter = apContainer.begin(behavior); iter.isValid(); ++iter) {
-    iter->touch();
-    if (behavior == autopas::IteratorBehavior::ownedOnly) {
-      EXPECT_TRUE(iter->isOwned());
-    } else if (behavior == autopas::IteratorBehavior::haloOnly) {
-      EXPECT_FALSE(iter->isOwned());
+  {
+    // with OpenMP:
+    auto begin = testRegionIterators ? apContainer.getRegionIterator(lowCorner, highCorner, behavior)
+                                     : apContainer.begin(behavior);
+    for (auto iter = begin; iter.isValid(); ++iter) {
+      iter->touch();
+      if (behavior == autopas::IteratorBehavior::ownedOnly) {
+        EXPECT_TRUE(iter->isOwned());
+      } else if (behavior == autopas::IteratorBehavior::haloOnly) {
+        EXPECT_FALSE(iter->isOwned());
+      }
     }
   }
```

Comment:
should we move the `#endif` to the end of the block? If `omp` is not defined we test the same thing twice.